backend/app/assets/stylesheets/spree/backend/shared/_forms.scss

```diff
@@ -306,3 +310,28 @@ fieldset {
 .form-buttons {
   text-align: center;
 }
+
+.form-section {
+  @include make-row(0px);
+  margin: 10px 0px;
+  padding: 10px 0px;
+  background: very-light($color-3, 4);
+  border: 1px solid very-light($color-3, 32);
+}
```

Comment:
The use of this class closely resembles the `.card` [component from Bootstrap](https://v4-alpha.getbootstrap.com/components/card/#header-and-footer). Do you think it will be an idea to either override the default `.card` class with our background and border, or maybe add a supplementary class to it, like:

```css
.card.form-section {
  background: very-light($color-3, 4);
  border: 1px solid very-light($color-3, 32);
}
```